trillian/ctfe/ct_server/main.go

```diff
@@ -276,7 +278,13 @@ func setupAndRegister(ctx context.Context, client trillian.TrillianLogClient, de
 	}
 	if *quotaRemote {
 		glog.Info("Enabling quota for requesting IP")
-		opts.RemoteQuotaUser = realip.FromRequest
+		opts.RemoteQuotaUser = func(r *http.Request) string {
+			var remoteUser = realip.FromRequest(r)
+			if remoteUser == "" {
```

Comment:
how about `len(remoteUser) == 0`? 
Just thinking that'd also catch `nil` if the library were ever to return it...

Reply:
SGTM. Done.